cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ExcludedBrokersForReplicaMoveTest.java

```diff
@@ -350,8 +408,22 @@ public void test() throws Exception {
                                  Collection<Integer> deadBrokers,
                                  Boolean expectedToOptimize,
                                  Boolean expectedToGenerateProposals) throws Exception {
+    return params(tid, goalClass, excludedBrokersForReplicaMove, exceptionClass, clusterModel,
+                  deadBrokers, expectedToOptimize, expectedToGenerateProposals, null);
+  }
+
+  private static Object[] params(int tid,
+      Class<? extends Goal> goalClass,
+      Set<Integer> excludedBrokersForReplicaMove,
+      Class<? extends Throwable> exceptionClass,
+      ClusterModel clusterModel,
+      Collection<Integer> deadBrokers,
+      Boolean expectedToOptimize,
+      Boolean expectedToGenerateProposals,
+      Properties configOverrides) throws Exception {
     deadBrokers.forEach(id -> clusterModel.setBrokerState(id, Broker.State.DEAD));
-    return new Object[]{tid, goal(goalClass), excludedBrokersForReplicaMove, exceptionClass, clusterModel, expectedToOptimize,
-                        expectedToGenerateProposals};
+    Goal goal = configOverrides == null ? goal(goalClass) : goal(goalClass, configOverrides);
+    return new Object[]{tid, goal, excludedBrokersForReplicaMove, exceptionClass, clusterModel, expectedToOptimize,
+        expectedToGenerateProposals};
```

Comment:
Bad indentation -- but we can completely drop this change via adding the relevant config property update to `goal` function directly. (please see earlier comment)

Reply:
Fixed the indentation issue. I prefer to keep the below change:
```java
configOverrides.put(AnalyzerConfig.TOPICS_WITH_MIN_LEADERS_PER_BROKER_CONFIG, TestConstants.TOPIC_MUST_HAVE_LEADER_REPLICAS_ON_BROKERS);
```
Because in some test cases, we want to set different values for the config `AnalyzerConfig.TOPICS_WITH_MIN_LEADERS_PER_BROKER_CONFIG` to test this new hard goal